src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,17 @@
+package seedu.addressbook.data.person;
+
+public class PostalCode{
+	private String postalCode;
```

Comment:
You can implement a method to validate the format of the postal code, e.g. must be numbers.